recipes-debian/glibc/glibc_debian.bbappend

```diff
@@ -23,3 +23,16 @@ do_install_append () {
 do_poststash_install_cleanup_append() {
     install -d ${D}/${libdir}/locale
 }
+
+FILES_${PN}-utils_append_class-nativesdk = "\
```

Comment:
meta/recipes-core/glibc/glibc-package.inc has the following line.

```
FILES_${PN}-utils = "${bindir}/* ${sbindir}/*"
```

I think this should also be used for nativesdk because meta-debian's glibc_debian.bb has `BBCLASSEXTEND = "nativesdk"`. Why isn't this applied for FILES_${PN}-utils?

Reply:
I checked  deby(poky with meta-debian)  that doesn't have same problems. It seems as if the glibc_debian.bbappend file changes package contents so that it needed to add FILES_${PN}-utils_append_class-nativesdk  block.

Reply:
Could you compare `FILES_nativesdk-glibc-utils` between deby and emlinux?

Reply:
@hiraku-wfs I've overlooked that this file removes some files from glibc-utils package. 

```
FILES_${PN}-utils_remove = "${sbindir}/*"
FILES_${PN}-utils += "${sbindir}/iconvconfig"
```
Thereby some of files are missing.

I'll re-create patch.